### DISCUSSION TITLE
Getting rid of clutter in callouts

### DISCUSSION THREAD

**Discussion by VanLyden** [c0nD/FluffyBOT#50] [Ideas]
![image](https://user-images.githubusercontent.com/115385250/219997114-1fba2efc-d68c-4841-a7f4-23c8ff1d7193.png)

This is just a suggestion, not something that needs to be fixed immediately, but is there any way to get rid of the message saying "Attempting to queue..." when someone tries to join the queue? I feel that it sort of clutters the callout channel and doesn't serve much purpose when the following message states the member that has joined the queue.

**Comment by c0nD** [comment]
This has been on the radar, and Issue #47 is going to be resolved this coming update

**Comment by c0nD** [comment]
Issue has now been solved.

Thread locked.
